tools/clang/unittests/HLSL/PixTest.cpp

```diff
@@ -195,10 +200,6 @@ class PixTest {
   TEST_METHOD(DiaCompileArgs)
   TEST_METHOD(PixDebugCompileInfo)
 
-  TEST_METHOD(CheckSATPassFor66_NoDynamicAccess)
-  TEST_METHOD(CheckSATPassFor66_DynamicFromRootSig)
-  TEST_METHOD(CheckSATPassFor66_DynamicFromHeap)
```

Comment:
Curious why these had to go

Reply:
Oh i should have dropped a comment: PIX doesn't use this functionality anymore, so I deleted these tests.